### DISCUSSION TITLE
[Keyball44] ipad proに接続しても動作しない

### DISCUSSION THREAD

**Discussion by lckisaac** [Yowkees/keyball#214] [Q&A] [answered]
お世話になります。先日Keyball44を完成し、利用させていただいております。
自宅ではノートパソコン(Windows)に繋げて、一部の作業・外出時にはipad pro(第二世代、11 in.)に繋げて運用したいと考えております。ノートパソコンに接続しているときは特に問題なく動作しておりますが、試しにipad proに繋げてみるとbacklight LEDのみ点灯し、動作できず困っております。(両方のpromicroで接続しても同じ挙動になります。)
他の利用者の方とも相談し解決に至らなかったのでこちらで質問させていただいた次第です。(大体の内容についてはこちらをご参照ください。https://twitter.com/lckisaac/status/1645213731067150336?s=20)
助言いただけますと幸いです。

**Comment by lckisaac** [comment]
こちらほぼ自己解決しました。
特になにもしてませんでしたが先日突然普通に繋がるようになってました。今日また動作しなくなってましたが、今度はSPLIT_USB_TIMEOUTの伸ばすことで一応無事に動くようになりました。お騒がせしました。

**Comment by Yowkees** [comment, marked as answer]
ご連絡が遅くなり申し訳ありませんでした。
自己解決とのことで安心しました。
当方、ipadで使用したことがないので原因は分かりませんがSPLIT_USB_TIMEOUTが関係するのですね。勉強になりました。

**Reply by lckisaac** [reply]
現時点で再発はしていないので原因は未だに特定できていないままです、そのためSPLIT_USB_TIMEOUTとの関係性は不明です(一回目の動作不能時の症状と二回目とは違うため)
考察になりますが、ipad proの使用年数によってバッテリーの劣化に伴ってUSBハブ経由でパソコンに接続した時のようにkeyballへの給電が不安定(低下気味？)になる現象が発生したと思われます。
